src/app/forms/validators/custom-validators.ts

```diff
@@ -266,4 +268,72 @@ export class CustomValidators {
 
     return !isZNumber ? null : { notZNumber: true };
   };
+
+  static handlePsvClassChange = (): ValidatorFn => {
+    return (control: AbstractControl): ValidationErrors | null => {
+      const vehicleSizeControl = control.root.get('techRecord_vehicleSize');
+      if (control.dirty) {
+        switch (control.value) {
+          case VehicleClass.DescriptionEnum.LargePsvIeGreaterThan23Seats: {
+            vehicleSizeControl?.setValue(VehicleSizes.LARGE, { emitEvent: false });
+            return null;
+          }
+          case VehicleClass.DescriptionEnum.SmallPsvIeLessThanOrEqualTo22Seats: {
+            vehicleSizeControl?.setValue(VehicleSizes.SMALL, { emitEvent: false });
+            return null;
+          }
+        }
+      }
+      return null;
+    };
+  };
+
+  static handlePsvSizeChange = (): ValidatorFn => {
+    return (control: AbstractControl): ValidationErrors | null => {
+      const vehicleClassControl = control.root.get('techRecord_vehicleClass_description');
+      if (control.dirty) {
+        switch (control.value) {
+          case VehicleSizes.LARGE: {
+            vehicleClassControl?.setValue(VehicleClass.DescriptionEnum.LargePsvIeGreaterThan23Seats, { emitEvent: false });
+            return null;
+          }
+          case VehicleSizes.SMALL: {
+            vehicleClassControl?.setValue(VehicleClass.DescriptionEnum.SmallPsvIeLessThanOrEqualTo22Seats, { emitEvent: false });
+            return null;
+          }
+        }
+      }
+      return null;
+    };
+  };
+
+  static handlePsvPassengersChange = (passengersOne: string, passengersTwo: string): ValidatorFn => {
+    return (control: AbstractControl): ValidationErrors | null => {
+      if (control.dirty) {
+        const controlOne: number = control.root.get(passengersOne)?.value;
+        const controlTwo: number = control.root.get(passengersTwo)?.value;
+        const controlThree: number = control.value;
+
+        const classControl = control.root.get('techRecord_vehicleClass_description');
+        const sizeControl = control.root.get('techRecord_vehicleSize');
+
+        const totalPassengers = controlOne + controlTwo + controlThree;
+
+        switch (true) {
+          case totalPassengers <= 22: {
+            sizeControl?.setValue(VehicleSizes.SMALL, { emitEvent: false });
+            classControl?.setValue(VehicleClass.DescriptionEnum.SmallPsvIeLessThanOrEqualTo22Seats, { emitEvent: false });
+            return null;
+            break;
```

Comment:
```suggestion
```
I think this break is unreachable

Reply:
You're right, I added the return in because I was having an issue and forgot to remove them